client/src/features/versions/versions.api.ts

```diff
@@ -113,29 +113,24 @@ export const versionsApi = createApi({
         if (response.versions?.length < 1)
           throw new Error("Unexpected response");
         const singleVersion = response.versions[0];
+
         return {
           name: response.name,
           version: singleVersion?.version ?? "unavailable",
           anonymousSessionsEnabled:
             singleVersion?.data?.anonymousSessionsEnabled ?? false,
           sshEnabled: singleVersion?.data?.sshEnabled ?? false,
-          cloudStorageEnabled: {
-            s3: singleVersion?.data?.cloudstorageEnabled?.s3 ?? false,
-            azureBlob:
-              singleVersion?.data?.cloudstorageEnabled?.azure_blob ?? false,
-          },
-        };
+          cloudStorageEnabled:
+            singleVersion?.data?.cloudstorageEnabled ?? false,
+        } as NotebooksVersion;
```

Comment:
nitpick, Should not be necessary:

```suggestion
        };
```